HACK.TXT

```diff
@@ -11,7 +11,8 @@ Capstone source is organized as followings.
 │   ├── Sparc       <- Sparc engine
 │   ├── SystemZ     <- SystemZ engine
 │   ├── X86         <- X86 engine
-│   └── XCore       <- XCore engine
+│   ├── XCore       <- XCore engine
+│   └── M680X       <- M680X engine
```

Comment:
this is in alphabet order. can you put this line before M68K?

Reply:
Done